### ISSUE TITLE
Mac Not Rendering

### ISSUE THREAD

**Issue by kennebel** [VirtualWhirlwind/vscode-povray2#8]
As reported on the news.povray.org site, pushing the button to render is not working. Need help (no access to mac environment) to figure out what a good command line looks like so I can replicate that in code.

**Comment by bentorkington** [comment]
I could check it out. Could you link to the post on news.povray.org?

**Comment by bentorkington** [comment]
Works for me after setting `povray.render.pvenginePath`. The plugin should probably check the existence of that setting because if it is empty it ends up passing the name of the .pov file to the shell, which fails to execute it:

```
/usr/local/bin/bash: /Users/ben/repos/me/retaining-wall/house.pov: Permission denied
The terminal process "/usr/local/bin/bash '-c', ' /Users/ben/repos/me/retaining-wall/house.pov Width=1440 Height=1080 Output_File_Name=/Users/ben/repos/me/retaining-wall/output/ +A'" terminated with exit code: 126.
```

Hope this helps

**Comment by kennebel** [comment]
Good idea of not trying to render if the field isn't filled in. I can pop a message, guiding the user to go fill it in. Alternately, i guess i could fall back on one existing in the path, but that seems troublesome, especially if a person has more than one version installed.

POV-Ray News link that reported the Mac error:
https://news.povray.org/povray.programming/message/%3Cweb.622bb69c8de14cd834c7846e6830a892%40news.povray.org%3E/#%3Cweb.622bb69c8de14cd834c7846e6830a892%40news.povray.org%3E

**Comment by bentorkington** [comment]
Look like that user gets their Povray from some app bundle installed in /Applications, but can't tell from the screenshot what the error is. My Povray is in /usr/local/bin which is where Homebrew installs it, but I couldn't guess what the most popular choice would be among Mac Povray users.

**Comment by pvaibhav** [comment]
On an M1 Mac the path would be `/opt/homebrew/bin/povray`

I've already configured this path in the extension settings, yet rendering doesn't work. It just pops up the command line string as a message in VScode bottom right corner.

If I copy-paste the exact string into the terminal, the render succeeds.

Any ideas?

**Comment by bentorkington** [comment]
Does a new terminal titled "Render Scene" open within Code? Even with an incorrect Povray path, I at least get a new terminal window where I can see any errors associated with launching Povray.

If so, please post any error messages from that terminal.
